lib/gooddata/lcm/actions/help.rb

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2010-2018 GoodData Corporation. All rights reserved.
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+require 'gooddata/bricks/pipeline'
+
+require_relative 'base_action'
+
+module GoodData
+  module LCM2
+    # Action responsible for printing bricks help
+    class Help < BaseAction
+      DESCRIPTION = 'Print Help Message'
+
+      PARAMS = {}
+
+      class << self
+        BRICK_PIPELINE_SUFFIX = /_brick_pipeline$/
+
+        def create_help
+          "Available bricks:" +
+            GoodData::Bricks::Pipeline.methods
+              .grep(BRICK_PIPELINE_SUFFIX)
+              .map { |method| "\n\t" + method.to_s.sub(BRICK_PIPELINE_SUFFIX, '') }.join
+        end
+
+        def call(params)
```

Comment:
warning: Lint/UnusedMethodArgument: Unused method argument - `params`. If it's necessary, use `_` or `_params` as an argument name to indicate that it won't be used. You can also write as `call(*)` if you want the method to accept any arguments but don't care about them. (rubocop)